tutorial/lesson_15_generators_usage.sh

```diff
@@ -198,30 +198,15 @@ check_no_runtime()
     target=host-x86-64-avx-no_runtime
 
 # These files don't contain the runtime
-check_no_runtime my_first_generator_basic.o
-check_symbol     my_first_generator_basic.o my_first_generator_basic
-check_no_runtime my_first_generator_sse41.o
-check_symbol     my_first_generator_sse41.o my_first_generator_sse41
-check_no_runtime my_first_generator_avx.o
-check_symbol     my_first_generator_avx.o my_first_generator_avx
+check_no_runtime my_first_generator_basic.a
+check_symbol     my_first_generator_basic.a my_first_generator_basic
+check_no_runtime my_first_generator_sse41.a
+check_symbol     my_first_generator_sse41.a my_first_generator_sse41
+check_no_runtime my_first_generator_avx.a
+check_symbol     my_first_generator_avx.a my_first_generator_avx
 
 # We can then use the generator to emit just the runtime:
 ./lesson_15_generate -r halide_runtime_x86 -o . target=host-x86-64
-check_runtime halide_runtime_x86.o
```

Comment:
The bit that was deleted is the motivation for the preceding steps... I know we're going to handle multi-target support differently in the future, but I'd like to preserve this bit even then, and follow it with a section explaining that you can get Halide to do this automatically by...